src/presets.asm

```diff
@@ -296,10 +405,19 @@ if !RAW_TILE_GRAPHICS
 else
     JSL $82E7D3  ; Load level data, CRE, tile table, scroll data, create PLMs and execute door ASM and room setup ASM
 endif
+    JSL preset_scroll_fixes
+
+    LDA !sram_preset_options : BIT !PRESETS_CLOSE_BLUE_DOORS : BNE .done_opening_doors
+    LDA !SAMUS_POSE : BEQ .done_opening_doors
```

Comment:
This only covers presets without any suits. $009B is the pose for facing forward with Varia or Gravity.